platforms/defaults_windows.go

```diff
@@ -41,6 +45,7 @@ func DefaultSpec() specs.Platform {
 type windowsmatcher struct {
 	specs.Platform
 	osVersionPrefix string
+	osUBR           int
```

Comment:
In what circumstances would `osUBR` not be the same as the fourth part of OSVersion? I realise the pattern exists, but I can't work out why we have `osVersionPrefix` either.

I also don't see `osUBR` being set, I assume it should be set in `newDefaultMatcher` like `osVersionPrefix` is, by splitting from `OSVersion`.